### ISSUE TITLE
args[module] warning for templated module parameter

### ISSUE THREAD

**Issue by netsandbox** [ansible/ansible-lint#2833]
##### Summary

The new `args[module]` rule reports warnings if you use a Jinja template for a module parameter.

##### Issue Type

- Bug Report

##### OS / ENVIRONMENT

```console (paste below)
ansible-lint --version
ansible-lint 6.10.0 using ansible 2.12.10
```

- ansible installation method: pip
- ansible-lint installation method: pip

##### STEPS TO REPRODUCE

```yaml
- name: Create directories on BurmillaOS
  ansible.builtin.file:
    path: "{{ item }}"
    state: "{{ certificates_enabled | ternary('directory', 'absent') }}"
    owner: root
    group: root
    mode: 0755
  loop:
    - "{{ certificates_cert_dir }}"
    - "{{ certificates_key_dir }}"
  when: ansible_facts.distribution == "BurmillaOS"
  tags: certificates
```

##### Desired Behavior

No Warning.

##### Actual Behavior

```shell
args[module]: value of state must be one of: absent, directory, file, hard, link, touch, got: {{ certificates_enabled | ternary('directory', 'absent') }} (warning)
```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve

**Comment by evgeni** [comment]
This is a documented limitation of the parser (see [args doc source](https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/args.md#special-cases) and [rendered args docs](https://ansible-lint.readthedocs.io/rules/args/#special-cases))

**Comment by netsandbox** [comment]
I really have to learn to not only read the warning, but also to read the documentation which is linked with the warning :man_facepalming: 
Sorry for the noise.
